versioned_docs/version-23.04/connect/saml.md

```diff
@@ -0,0 +1,130 @@
+  user's full name.
+
+</TabItem>
+<TabItem value="Users manual management" label="Manual management">
+
+On page **Configuration > Users > Contacts/Users**, [create the users](../monitoring/basic-objects/contacts-create.md)
+that will log on to Centreon using SAML.
+
+</TabItem>
+</Tabs>
+
+### Step 5: Manage Authorizations
+
+<Tabs groupId="sync">
+<TabItem value="Role automatic management" label="Automatic management">
+
+If you turn on **Enable automatic management**, users that log in to Centreon will be automatically
+  [granted rights](../administration/access-control-lists.md), as they will be linked to
+  [access groups](../administration/access-control-lists.md#creating-an-access-group) according to the rules you have defined.
+  
+- **Apply only first role**: If several roles are found for a specific user in the identity provider's information, then
+  only the first role will be applied. If the option is turned off, all roles will be applied.
+
+- Define which attribute from which endpoint will be used to retrieve values to create relationships with access groups.
+- Match the attributes retrieved from the identity provider with the contact groups you want the user to belong to.
+
+> Each time the user logs in, authorization management is reinitialized to take into account any new information from the
+> identity provider.
+
+</TabItem>
+<TabItem value="Role manual management" label="Manual management">
+
+If you turn off **Enable automatic management**, you have to [grant users rights](../administration/access-control-lists.md)
+manually by linking them to [access groups](../administration/access-control-lists.md#creating-an-access-group).
+
+</TabItem>
+</Tabs>
+
+### Step 6: Manage Contact groups
+
+<Tabs groupId="sync">
+<TabItem value="Groups automatic management" label="Automatic management">
+
+If you turn on **Enable automatic management**, users that log in to Centreon will be attached to the
+[contact groups](../monitoring/basic-objects/contacts-groups.md#contact-groups) you have defined.
+
+- Define which attribute from the identity provider will be used to retrieve values to create relationships with access groups.
+- Match the attributes retrieved from the identity provider with the contact groups you want the user to belong to.
+
+> Each time the user logs in, groups management is reinitialized to take into account any new information from the identity provider.
+
+</TabItem>
+<TabItem value="Groups manual management" label="Manual management">
+
+If you turn off **Enable automatic management**, you have to manage manually the relation between contact and
+[contact groups](../monitoring/basic-objects/contacts-groups.md#contact-groups).
+
+</TabItem>
+</Tabs>
+
+### Step 7: Configure your Identity Provider (IdP)
+
+Configure your Identity Provider to add the Centreon application to use the SAML protocol to authenticate your users:
```

Comment:
```suggestion
Configure your identity provider so that the Centreon application can use the SAML protocol to authenticate your users. Here is an example of fields you may have to fill in:
```